docs/pages/build-reference/ios-capabilities.mdx

```diff
@@ -13,7 +13,7 @@ When you make a change to your iOS entitlements, this change needs to be updated
 
 In the managed workflow, the entitlements are read from the introspected app config. To edit them, see the [`ios.entitlements`](/versions/latest/config/app/#entitlements) field in your app config file. You can see your introspected config by running `npx expo config --type introspect` in your project and then look for the `ios.entitlements` object for the results.
 
-In bare workflow apps, the entitlements are read from your __ios/**/*.entitlements__ file.
+In bare workflow apps, the entitlements are read from your **ios/\*_/_.entitlements** file.
```

Comment:
```suggestion
In bare workflow apps, the entitlements are read from your **ios/\*\*/\*.entitlements** file.
```

I think Prettier formatting broke this. After Prettier's formatting is applied, following is the result:

<img width="696" alt="CleanShot 2023-06-08 at 10 56 44@2x" src="https://github.com/expo/expo/assets/10234615/30754585-efd3-422a-8b47-cf26193ce50a">

The above suggestion fixes it.